api/src/main/resources/org/openmrs/module/pihmalawi/reporting/reports/sql/ic3_register.sql

```diff
@@ -234,8 +232,4 @@ SELECT
   CASE WHEN age >= 19 
     THEN round(lastWeight/POWER(lastHeight/100,2),1)
   END AS BMI
-FROM warehouseCohortTable;
-
```

Comment:
Also, we should support empty lines.  Should be easy to add into the code above.